Automattic/jetpack
# AI Assistant: generate content from selected blocks

Fixes #

## Proposed changes:

* https://github.com/Automattic/jetpack/issues/30547

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Ensure you have a few paragraphs there
* Select multiple blocks
* Confirm you see the multiple blocks edition in the sidebar 
* Generate new content 

https://github.com/Automattic/jetpack/assets/77539/5a03bdc8-c52c-4491-a751-f95897a624bb

* When the content is unclear, the app will try to show a notice:

<img width="879" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/60a3af91-72b5-4745-9a04-3a98301d02d0">
